frontend/public/components/utils/headings.tsx

```diff
@@ -167,6 +168,7 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
               'co-m-pane__heading--center': centerText,
               'co-m-pane__heading--logo': props.icon,
               'co-m-pane__heading--with-help-text': helpText,
+              'co-m-pane__heading--margin-right': rightMargin,
```

Comment:
Rather than adding a new prop further complicating an already complex component, it seems to me it's better to leave the margins on the `co-operator-details__toggle-value` since that's the only case that needs it.  Yea/nay?

Reply:
My reason for adding the right margin to the `<h1>` is to improve the left alignment when at mobile size.

<img width="699" alt="Screen Shot 2022-07-19 at 9 52 41 AM" src="https://user-images.githubusercontent.com/1874151/179767736-bb3448d8-9008-4b96-ad28-9a77faa5f27a.png">

----

<img width="702" alt="Screen Shot 2022-07-19 at 9 53 12 AM" src="https://user-images.githubusercontent.com/1874151/179767754-9a4c54b8-120c-40b7-8254-b07dcce0356c.png">